code/modules/projectiles/guns/projectile/automatic/bulldog.dm

```diff
@@ -4,7 +4,7 @@
 	icon = 'icons/obj/guns/projectile/bulldog.dmi'
 	icon_state = "bulldog"
 	item_state = "bulldog"
-	w_class = ITEM_SIZE_BULKY
+	w_class = ITEM_SIZE_NORMAL
```

Comment:
I frankly dont understand this change at all.
Its a massive buff to a descent gun that is out of circulation?
Dosnt matter to much I suppose

Reply:
Error on my end it seems, I didn’t notice till now I reduced that one further. Don’t know if I copy-pasted the wrong value or what on that one. I’ll amend that.